src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.Linux.cs

```diff
@@ -251,6 +249,7 @@ private sealed class RunningInstance
                 _includeSubdirectories = includeSubdirectories;
                 _notifyFilters = notifyFilters;
                 _cancellationToken = cancellationToken;
+                FileSystemWatcher.CaseSensitive = true;
```

Comment:
This is ok, though what I'd had in mind was that each .Platform.cs file would still define its own CaseSensitive... in .Windows.cs and .Linux.cs it'd be a constant, and in .OSX.cs it'd be a property.  That way, for the systems where it's a fixed constant, call sites could optimize appropriately.  Not a big deal, though.